src/server/routes/units.js

```diff
@@ -165,7 +166,7 @@ router.post('/addUnit', async (req, res) => {
 	const validationResult = validate(req.body, validUnit);
 	if (!validationResult.valid) {
 		log.error(`Invalid input for unitsAPI. ${validationResult.error}`);
-		res.sendStatus(400);
+		failure(res, 400, 'Validation failed with  ' + validationResult.error.toString());
```

Comment:
Could this failure include that it is for adding a unit? I would agree that the original log message right above it could be more specific (not code you did).